device/nokia/armhf-nokia_ixs7215_52x-r0/installer.conf

```diff
@@ -1 +1,10 @@
 VAR_LOG_SIZE=4096
+
+if [ -r /host/machine.conf ]; then
+  # fstrim frequency to daily for our platform
+  desc="Discard unused blocks ($platform)"
+  sed -i -e "s,OnCalendar=.*,OnCalendar=daily,g" \
+         -e "s,Description=.*,Description=$desc,g" \
+         /usr/lib/systemd/system/fstrim.timer
```

Comment:
Instead of modifying the packaged file, please create an override file for this timer, in `/etc/systemd/system/fstrim.timer.d/`. This is far cleaner and less likely to break something in the case of some updates/changes.

Reply:
Will do.  That is much cleaner.  Thank you.